src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -462,7 +462,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                 _lastHoveredId = newId;
                 _lastHoveredInterval = newInterval;
                 _renderEngine->UpdateHyperlinkHoveredId(newId);
-                _renderer->UpdateLastHoveredInterval(newInterval);
+                _renderEngine->UpdateLastHoveredInterval(newInterval);

```

Comment:
This is a fantastic example of how this PR decouples `renderer.cpp` & `DxEngine`. 

Only `DxEngine` actually needs `UpdateLastHoveredInterval`. `UpdateLastHoveredInterval` never should be inside `renderer.cpp`.